Assignment 2/Assignment-2/Validator.swift

```diff
@@ -0,0 +1,68 @@
+//
+//  Validator.swift
+//  Assignment-2
+//
+//  Created by Shubham on 26/01/22.
+//
+
+import Foundation
+
+
+func nameValidator(name : String?) ->Bool{
+    if(name == nil){
+        print("wrong name entered")
+        return false
+    }
+    if(name!.isEmpty == true){
```

Comment:
merge this and the above if , since both do the same task , no need of having 2 different if , make use of suitable logical operator

Reply:
okay done.